WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -1155,10 +1151,13 @@ - (UIBarButtonItem*)cancelChevronButton
         return _cancelChevronButton;
     }
     
-    WPButtonForNavigationBar* cancelButton = [self buttonForBarWithImageNamed:@"icon-posts-editor-chevron"
-                                                                        frame:NavigationBarButtonRect
-                                                                       target:self
-                                                                     selector:@selector(cancelEditing)];
+    UIImage *image = [UIImage imageNamed:@"icon-posts-editor-chevron"];
```

Comment:
I guess that extra size is padding? At the moment there's no way to do that with the framework, but it would definitely be nice to be able to specify insets / padding for an image.